sonar-core/src/main/resources/org/sonar/l10n/core.properties

```diff
@@ -3,7 +3,6 @@
 # GENERIC WORDS, sorted alphabetically
 #
 #------------------------------------------------------------------------------
-
```

Comment:
A large number of empty lines was removed in this file. I just want to make sure, that it was on purpose.

Reply:
good point, empty lines should not have been removed